### PR TITLE
Delete mention of enabling the casm explicitly

### PR DESCRIPTION
**Pull request by Arcticae**

### CODE REVIEW

**Review by piotmag769 — APPROVED** [review]

**Comment by MaksymilianDemitraszek** [comment]
Have we remove this from the template?

**Comment by MaksymilianDemitraszek** [comment]
image
We also could probably remove it from our codebase.

**Comment by piotmag769** [comment]
@MaksymilianDemitraszek enabling casm in Scarb.toml is fine, it's just not necessary - if the casm is present we will use it, if not, we will use USC

**Comment by MaksymilianDemitraszek** [comment]
Imo would be better to have one way to do this.

**Comment by piotmag769** [comment]
>Don't see any reason why we would want to consume scarb produced casm.

It is embedded in our logic though (in scarb-api I believe), I remember that I supported this idea. If Scarb already produced casm why should we recompile the sierra?

File: crates/sncast/tests/utils/build_contracts.sh [hunk 1]
```diff
@@ -15,10 +15,22 @@ if [ ! -d "${REPO_ROOT}/crates/sncast/tests/utils/devnet/bin" ]; then
   exit 1
```
Comment (THenry14) [inline crates/sncast/tests/utils/build_contracts.sh:15]: this file will be removed in https://github.com/foundry-rs/starknet-foundry/pull/2008 as it is no longer needed

**Review by THenry14 — CHANGES_REQUESTED** [review]
Do you mind giving some more context? what is this change related to?

**Comment by piotmag769** [comment]
>Do you mind giving some more context? what is this change related to?

@THenry14 You don't need the `casm = true` in Scarb since we can compile with USC

https://github.com/foundry-rs/starknet-foundry/blob/master/crates/scarb-api/src/lib.rs#L55

**Comment by THenry14** [comment]
> > Do you mind giving some more context? what is this change related to?
> 
> @THenry14 You don't need the `casm = true` in Scarb since we compile contracts with USC when needed
> 
> https://github.com/foundry-rs/starknet-foundry/blob/master/crates/scarb-api/src/lib.rs#L55

ahh, thanks. this is the context I was missing 🙏

**Pull request merged**